packages/client/tests/functional/jest.config.js

```diff
@@ -2,7 +2,7 @@
 const os = require('os')
 const path = require('path')
 
-const runtimeDir = path.dirname(require.resolve('../../runtime'))
+const runtimeDir = path.dirname(require.resolve('../../runtime/library'))
```

Comment:
Wondering if this is important: In CI we also run all the tests with these env vars
```
    PRISMA_CLI_QUERY_ENGINE_TYPE: binary
    PRISMA_CLIENT_ENGINE_TYPE: binary
```

So it feels wrong that the `runtimeDir` is set to `library` in the case CI runs with `binary`?

Reply:
Right, I also wonder if this has any impact on `getTestSuiteClient.ts` since we pass runtime paths there, maybe not.